SU-SOE/stanford_soe_helper
# SOE-2194: Related Articles: retain three-across

# READY FOR REVIEW

# Summary

1.  Magazine Node Related Articles: retain three-across for organization until 581px wide, then jump to single column. See https://sites.stanford.edu/jse-soe/magazine/article/new-semiconductor-materials-exceed-some-silicon-s-secret-powers (Never have two in a row)
2. Magazine grid pages: retain two-column approach for Related Issues section until apr. 400px  https://sites.stanford.edu/jse-soe/magazine
3. Main masthead gap @ 767px: see very faint light gray gap that opens in between the SU red header bar and the main masthead
4. In the footer, the signup button breaks to a new line


# Needed By (Date)
- 

# Criticality
- 

# Steps to Test
1. Verify never have two in a row: Go to /magazine/article/new-semiconductor-materials-exceed-some-silicon-s-secret-powers 
2. Magazine Related Issues retain two-column approach for until apr. 400px https://sites.stanford.edu/jse-soe/magazine
3. Check that the Main masthead gap @ 767px, very faint light gray gap that opens in between the SU red header bar and the main masthead is gone.
4. This was done in a previous ticket.
# Affects 
- SoE

# Associated Issues and/or People
## Related JIRA ticket(s)
- https://stanfordits.atlassian.net/browse/SOE-2194
## Related PRs

## More Information
https://invis.io/WDAOBEJ4P#/230196822_Section_Landing_L-f_04
## Folks to notify


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
